src/actions/actions.ts

```diff
@@ -2246,6 +2249,48 @@ class MoveToMatchingBracket extends BaseMovement {
   }
 }
 
+abstract class MoveInsideCharacter extends MoveToMatchingBase {
+  modes = [ModeName.Normal, ModeName.Visual, ModeName.VisualLine];
+  protected charToMatch: string;
+
+  public async execAction(position: Position, vimState: VimState): Promise<Position | IMovement> {
+    const text = TextEditor.getLineAt(position).text;
+    const pairedCharacter = this.pairings[this.charToMatch]
+    const charToClose = pairedCharacter ? pairedCharacter.match : this.charToMatch;
+
+    // First, search backwards for the opening character of the sequence
+    let start = -1;
```

Comment:
Is there a more idiomatic way to accomplish this without looping backwards? It feels weird using these -1 sentinel values, and using a plain for loop, but String.indexOf can only search forwards.